tower_modules/tower_settings/tasks/main.yml

```diff
@@ -1,14 +1,36 @@
 - name: Set the value of AWX_PROOT_SHOW_PATHS
   tower_settings:
     name: "AWX_PROOT_SHOW_PATHS"
-    value: "'/var/lib/awx/projects/', '/tmp'"
+    value:
+      - '/var/lib/awx/projects/'
+      - '/tmp'
+  register: result
+
+- assert:
+    that:
+      - "result is changed"
```

Comment:
In order to have changed as result we need to make sure that these values were not there. That is why we are reverting it on the next task. Another approach would be checking if the result is changed or ok.